pgml-dashboard/static/css/scss/base/_typography.scss

```diff
@@ -11,10 +13,60 @@
 // .h5 {font-weight: 700; font-size: 28px; line-height: 34px;}
 // .h6 {font-weight: 700; font-size: 24px; line-height: 30px;}
 
-.subcopy-text {font-size: 18px; line-height: 22px}
-.body-text {font-size: 16px; line-height: 20px}
-.legal-text {font-size: 12px; line-height: 16px;}
+.subcopy-text {
+    font-size: 18px;
+    line-height: 22px;
+}
+.body-text {
+    font-size: 16px;
+    line-height: 20px;
+}
+.legal-text {
+    font-size: 12px;
+    line-height: 16px;
+}
 
-.text-black {color: #{$gray-900} !important;}
-.text-white {color: #{$gray-100} !important;}
-.text-soft-white { color: #{$gray-200} !important; }
+.text-black {
+    color: #{$gray-900} !important;
+}
+.text-white {
+    color: #{$gray-100} !important;
+}
+.text-gradient-blue {
```

Comment:
Using a mixin for this would prevent the repetition.

Reply:
Let me update this. I haven't used mixins before

Reply:
classes are better than mixins, mixings generate a ton of repetitive css code, while classes inherit and cascade.

Reply:
@extend is also an option that minimizes css code (same as having two classes) and reduces the number of classes an element needs to take.  If we do go with two classes here we should have the _typographic class .text-gradient then a utility class for all the gradients so they can be applies as text color, background color, border color ect.